runtime/src/impls.rs

```diff
@@ -72,324 +72,3 @@ impl WeightToFeePolynomial for WeightToFee {
-			assert!(next < fm);
-		});
-
-		run_with_system_weight(target() / 2, || {
-			let next = runtime_multiplier_update(fm);
-			assert_eq_error_rate!(
-				next,
-				truth_value_update(target() / 2 , fm),
-				Multiplier::from_inner(100),
-			);
-			// Light block. Multiplier is reduced a little.
-			assert!(next < fm);
-
-		});
-		run_with_system_weight(target(), || {
-			let next = runtime_multiplier_update(fm);
-			assert_eq_error_rate!(
-				next,
-				truth_value_update(target(), fm),
-				Multiplier::from_inner(100),
-			);
-			// ideal. No changes.
-			assert_eq!(next, fm)
-		});
-		run_with_system_weight(target() * 2, || {
-			// More than ideal. Fee is increased.
-			let next = runtime_multiplier_update(fm);
-			assert_eq_error_rate!(
-				next,
-				truth_value_update(target() * 2 , fm),
-				Multiplier::from_inner(100),
-			);
-
-			// Heavy block. Fee is increased a little.
-			assert!(next > fm);
-		});
-	}
-
-	#[test]
-	fn weight_mul_grow_on_big_block() {
-		run_with_system_weight(target() * 2, || {
-			let mut original = Multiplier::zero();
-			let mut next = Multiplier::default();
-
-			(0..1_000).for_each(|_| {
-				next = runtime_multiplier_update(original);
-				assert_eq_error_rate!(
-					next,
-					truth_value_update(target() * 2, original),
-					Multiplier::from_inner(100),
-				);
-				// must always increase
-				assert!(next > original, "{:?} !>= {:?}", next, original);
-				original = next;
-			});
-		});
-	}
-
-	#[test]
-	fn weight_mul_decrease_on_small_block() {
-		run_with_system_weight(target() / 2, || {
-			let mut original = Multiplier::saturating_from_rational(1, 2);
-			let mut next;
-
-			for _ in 0..100 {
-				// decreases
-				next = runtime_multiplier_update(original);
-				assert!(next < original, "{:?} !<= {:?}", next, original);
-				original = next;
-			}
-		})
-	}
-
-	#[test]
-	fn weight_to_fee_should_not_overflow_on_large_weights() {
-		let kb = 1024 as Weight;
-		let mb = kb * kb;
-		let max_fm = Multiplier::saturating_from_integer(i128::max_value());
-
-		// check that for all values it can compute, correctly.
-		vec![
-			0,
-			1,
-			10,
-			1000,
-			kb,
-			10 * kb,
-			100 * kb,
-			mb,
-			10 * mb,
-			2147483647,
-			4294967295,
-			MaximumBlockWeight::get() / 2,
-			MaximumBlockWeight::get(),
-			Weight::max_value() / 2,
-			Weight::max_value(),
-		].into_iter().for_each(|i| {
-			run_with_system_weight(i, || {
-				let next = runtime_multiplier_update(Multiplier::one());
-				let truth = truth_value_update(i, Multiplier::one());
-				assert_eq_error_rate!(truth, next, Multiplier::from_inner(50_000_000));
-			});
-		});
-
-		// Some values that are all above the target and will cause an increase.
-		let t = target();
-		vec![t + 100, t * 2, t * 4]
-			.into_iter()
-			.for_each(|i| {
-				run_with_system_weight(i, || {
-					let fm = runtime_multiplier_update(max_fm);
-					// won't grow. The convert saturates everything.
-					assert_eq!(fm, max_fm);
-				})
-			});
-	}
-}
```

Comment:
Why remove all these tests?

Reply:
We are not maintaining the code for this. I would delegate this to parity for the Fee multiplier testing. These are orphaned tests